documentation/scope-commands.md

```diff
@@ -0,0 +1,65 @@
+# Scope Command Reference
+Below is a detailed list outlining all of the available scope commands, including what they do, how to use them, and information on how they fit into the workflow of an air traffic controller.
+
+Other than moving the data block or accepting handoffs, most commands require that the radar target be under your scope's control.
+
+Please note that the bracketed words (and brackets themselves) used in the syntax examples should be replaced with the appropriate data. Also note that `CID` was used in the examples, but aircraft may be referenced with any of (a) the CID, (b) the full callsign, or (c) the squawk code.
```

Comment:
Can you reference by CID?  I don't remember if we made that a reliable property on the aircraft.

Reply:
guess my comment here didn't post? hm. anyway, we cannot. So I scratched those pieces out.